### PR TITLE
Rcar sd mmc fixes

### PR DESCRIPTION
**Pull request by xakep-amatop**
Set power-source property for MMC pins for h3ulcb (CA57) and Salvator-XS boards.
Add MMC node to DTS and describe MMC pins for R-Car Spider S4 (CA55).
Add SD node to DTS and describe SD pins for h3ulcb (CA57).
Add overlay file for testing `tests/subsys/sd/sdmmc` on h3ulcb (CA57) board.

### CODE REVIEW

File: tests/drivers/disk/disk_performance/boards/rcar_spider_s4_r8a779f0_a55.overlay
Comment (danieldegrasse) [inline tests/drivers/disk/disk_performance/boards/rcar_spider_s4_r8a779f0_a55.overlay:1]: Any reason not to just set these properties at the board level? Enabling the SD controller/MMC device by default should not pull additional code into the image unless `CONFIG_DISK=y`, in which case I expect the user would want the MMC to be enabled.


Same comment applies to the other board overlay in this PR

**Comment by decsny** [comment]
please provide a PR description or I will not review

**Review by danieldegrasse — APPROVED** [review]

File: tests/drivers/disk/disk_performance/boards/rcar_spider_s4_r8a779f0_a55.overlay
Comment (xakep-amatop) [reply tests/drivers/disk/disk_performance/boards/rcar_spider_s4_r8a779f0_a55.overlay:1]: done and tested changes, thank you

**Comment by xakep-amatop** [comment]
> please provide a PR description or I will not review

Done

**Comment by danieldegrasse** [comment]
@decsny can you take another look?

**Review by decsny — APPROVED** [review]

**Pull request merged**